src/test/java/uk/gov/companieshouse/AppealApplicationTests.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.companieshouse;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Map;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+/**
+ * Basic integration tests
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class AppealApplicationTests {
```

Comment:
To be honest I planned to have no tests in the scope of this task and select testing framework as next step. I am not necessarily saying to remove it right now but I am signalling that we might consider selection of best testing framework later in the process.